Engineering-Reference/M365DestinationAO.md

```diff
@@ -173,15 +171,15 @@ Default Value: 180000
 
 The maximum period of time used to check if a OneDrive site has been provisioned.
 
-Change Conditions: ??
+Change Conditions: Timeout value can be changed. Can cause performance issues, not recommended.
 
 ### Patch Permissions <a name="patchperm"></a>
 {: .no_toc }
 Default Value: None
 
 Add/Update/Delete permissions for existing items. If disabled, only new permissions will be applied to existing items (see documentation).
 
-Change Conditions: ??
+Change Conditions: When enabled all permissions will be completely reapplied when performing delta migrations.
```

Comment:
Thought the permissions would be updated if they are present or applied if permissions  are not there.   I don't think it reapplies them if they are present though.

Reply:
Nope it repatches all permissions when enabled